util/jurisdiction_bulk_update.py

```diff
@@ -1,7 +1,32 @@
+import csv
+import io
 import uuid
-from arlo_server.models import Election, JurisdictionAdministration, User, Jurisdiction  # type: ignore
-from typing import Tuple, List
 from sqlalchemy import func
+from typing import Tuple, List
+
+from arlo_server.models import Election, JurisdictionAdministration, User, Jurisdiction  # type: ignore
```

Comment:
Shouldn't need this after I merge https://github.com/votingworks/arlo/pull/351

Reply:
Cool, I removed this and others.